src/cargo/core/package.rs

```diff
@@ -104,6 +105,8 @@ pub struct SerializedPackage {
     metabuild: Option<Vec<String>>,
     default_run: Option<String>,
     rust_version: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    profiles: Option<TomlProfiles>,
```

Comment:
> Cargo only looks at the profile settings in the Cargo.toml manifest at the root of the workspace. Profile settings defined in dependencies will be ignored.

imo we should only serialize profiles at the workspace level so people get something more akin to expected cargo behavior